docs/source/alignment.rst

```diff
@@ -0,0 +1,68 @@
+**********************
+Verification Alignment
+**********************
+
+A forecast is verified by comparing a set of initializations at a given lead to
+observations over some window of time. However, there are a few ways to decide *which*
+initializations or verification window to use.
+
+One can pass the keyword ``alignment=...`` to any compute functions (e.g.,
+:py:func:`~climpred.prediction.compute_persistence`,
+:py:func:`~climpred.prediction.compute_hindcast`,
+:py:meth:`~climpred.classes.HindcastEnsemble.verify`) to change the behavior for
+aligning forecasts with the verification product.
+
+The available keywords are:
+
+*  ``'same_init'`` (current default): Use a common set of initializations that verify
+   across all leads. This ensures that there is no bias in the result due to the state
+   of the system for the given initializations.
+
+
+*  ``'same_verif'``: Use a common verification window across all leads. This ensures that
+   there is no bias in the result due to the observational period being verified
+   against.
+
+
+*  ``'maximize'``: Use all available initializations at each lead that verify against
+   the observations provided. This changes both the set of initializations and the
+   verification window used at each lead.
+
+Same initializations
+####################
+
+(**current default**)
+
+.. code::
+
+    ``alignment='same_inits'``
+
+Below is an example of the logic used in ``climpred`` to select initializations that
+verify with the given verification data over all leads. This type of alignment is used
+in [MurCSS]_.
```

Comment:
I will have to double check that. I didnt find this in the code. would you please remove this reference for now?